OloEditor/src/Panels/ContentBrowser/ContentBrowserItem.cpp

```
@@ -0,0 +1,371 @@
+        // Click handling
+        if (ImGui::IsItemHovered())
+        {
+            SetAction(result, ContentBrowserAction::Hovered);
+
+            if (ImGui::IsMouseDoubleClicked(ImGuiMouseButton_Left))
+            {
+                SetAction(result, ContentBrowserAction::Activated);
+            }
+            else if (ImGui::IsMouseClicked(ImGuiMouseButton_Left))
+            {
+                bool ctrl = ImGui::GetIO().KeyCtrl;
```

Clang (14.0.6)

[warning] variable 'ctrl' is not initialized

(cppcoreguidelines-init-variables)